mixer_paths.xml

```diff
@@ -10,7 +10,7 @@
     <ctl name="Voice Rx Gain" id="0" value="0" />
     <ctl name="Voice Rx Gain" id="1" value="-1" />
     <ctl name="Voice Rx Gain" id="2" value="20" />
-    <ctl name="TTY Mode" value="Off" />
+    <ctl name="TTY Mode" value="OFF" />
```

Comment:
Why this is needed?

Reply:
This will fix audio_hw_primary error cant define value Off on tty must be all caps.